bittide-instances/src/Bittide/Instances/Domains.hs

```diff
@@ -19,3 +25,34 @@ createDomain vXilinxSystem{vName="External", vPeriod=hzToPeriod 200e6}
 createDomain vXilinxSystem{vName="GthRx", vPeriod=hzToPeriod 125e6}
 createDomain vXilinxSystem{vName="GthTx", vPeriod= hzToPeriod 125e6}
 createDomain vXilinxSystem{vName="Internal", vPeriod=hzToPeriod 200e6}
+
+type CccBufferSize = 25 :: Nat -- Is this used?
+type CccStabilityCheckerMargin = 10 :: Nat
+type CccStabilityCheckerFramesize dom = PeriodToCycles dom (Seconds 2)
+type CccReframingWaitTime dom = PeriodToCycles dom (Seconds 5)
+
+instancesClockConfig ::
+  forall dom .
+  KnownDomain dom =>
+  Proxy dom ->
+  ClockControlConfig
+    dom
+    CccBufferSize
+    CccStabilityCheckerMargin
+    (CccStabilityCheckerFramesize dom)
+instancesClockConfig Proxy = ClockControlConfig
+  { cccPessimisticPeriod         = pessimisticPeriod
+  , cccPessimisticSettleCycles   = pessimisticSettleCycles self
+  , cccSettlePeriod              = microseconds 1
+  , cccStepSize                  = stepSize
+  , cccBufferSize                = SNat @CccBufferSize
```

Comment:
I removed the `@` annotations and added function documentation that details what is changed compared to `defClockConfig`. Is that okay?

Reply:
The ` CccReframingWaitTime` type is not used and differs from the waiting time that's set below. Also setting a waiting time has no effect, because reframing is disabled. :upside_down_face:

Overall, I am fine with that config, but it still looks to me like it was put together in a hurry.